testsuite/features/secondary/allcli_system_group.feature

```diff
@@ -106,7 +177,7 @@ Feature: Manage a group of systems
 
   # Red Hat-like minion is intentionally not removed from group
```

Comment:
I see this comment and I see the removed scenario above. Not sure what's up but we need to decide either way.

Reply:
That's precisely because of this comment that we have to remove the scenario above.

The idea was to test the removal of a group with still a minion in it (the RH-like minion, in this case).

Reply:
but that removes the minion from the group, doesn't delete the group

Reply:
Group is removed at line 113

Reply:
I don't understand this but if it's deliberate ok. Please remove the comment here as well if you're removing the scenario as well and ok